schemas/fingerprint-server-api.yaml

```diff
@@ -1023,9 +1085,77 @@ components:
-                    • `false` - No signs of emulated environment detected
-
-                    Available only for events from Android client. The field will not be present for a browser or iOS event.
+                    • `false` - No signs of emulated environment detected or the client is not Android.
+                  example: false
+            error:
+              $ref: '#/components/schemas/ProductError'
+        clonedApp:
+          title: SignalResponseClonedApp
+          type: object
+          properties:
+            data:
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    Android specific cloned application detection. There are 2 values:
+                    • `true` - Cloned application detected (e.g. fully cloned application with a slightly different package name)
+                    • `false` - No signs of cloned application detected or the client is not Android.
+                  example: false
+            error:
+              $ref: '#/components/schemas/ProductError'
+        factoryReset:
+          title: SignalResponseFactoryReset
+          type: object
+          properties:
+            data:
+              type: object
+              properties:
+                time:
+                  title: Time
+                  description: >
+                   Time in UTC for the Android client when recent factory reset was done. 
+                   If there is no sign of factory reset or the client isn't Android, the field will be epoch time.
+                  type: string
+                  format: date-time
+                  example: '2022-06-09T22:58:36Z'
+                timestamp:
+                  description: Same value as it's in the `time` field but represented in timestamp format.
+                  type: integer
+                  format: int64
+                  example: 1654815517198
+            error:
+              $ref: '#/components/schemas/ProductError'
+        jailbroken:
+          title: SignalResponseJailbroken
+          type: object
+          properties:
+            data:
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    iOS specific jailbreak detection. There are 2 values:
+                    • `true` - Jailbreak detected
+                    • `false` - No signs of jailbreak or the client is not iOS.
+                  example: false
+            error:
+              $ref: '#/components/schemas/ProductError'
+        frida:
+          title: SignalResponseFrida
+          type: object
+          properties:
+            data:
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    iOS specific frida detection. There are 2 values:
```

Comment:
Same question as above, also shouldn't `frida` be capitalized?

Reply:
done